presto-hive/src/main/java/com/facebook/presto/hive/metastore/MetastoreUtil.java

```diff
@@ -424,7 +424,8 @@ public static RoleGrant fromRolePrincipalGrant(RolePrincipalGrant grant)
 
     public static Set<String> listEnabledRoles(Identity identity, Function<PrestoPrincipal, Set<RoleGrant>> listRoleGrants)
     {
-        Optional<SelectedRole> role = identity.getRole();
+        // TODO identity.getrole()
```

Comment:
I can not. Because at this point Identity no longer has role within. It has a map of roles instead.